src/components/blocks/api-reference/headings/ApiReferenceH6.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ComponentProps, ReactNode } from 'react';
+import styled from 'styled-components';
+import GenericHtmlBlock from '../../Html/GenericHtmlBlock';
+
+// Preferring styled comopnents here because props pass through automatically.
+const StyledApiReferenceH6 = ({ attribs, children }: { attribs: ComponentProps<'h6'>; children: ReactNode }) => (
+  <h6
+    {...{ ...attribs }}
+    className={`${attribs?.className ?? ''} font-mono
+    items-center w-min pb-4 px-8 mt-10 mb-24
+    rounded-sm bg-api-reference-attribute-highlight border border-api-reference-attribute-border`}
+  >
+    {children}
+  </h6>
+);
+styled.h6`
+  align-items: center;
+  padding: 4px 6px;
+  background: #ffe6dc;
```

Comment:
if this color is the style guide, you can use css vars here `var(--color-x)`

Reply:
Sadly I don't think it is, I searched Ably UI and it wasn't there; I believe the API Reference design was intended to be a quick update of the old API Reference stylings to be more in keeping with the overall new design, and hasn't been integrated fully with the linear design system yet.